.github/workflows/ci.yml

```diff
@@ -67,5 +67,8 @@ jobs:
       - name: Build typescript
         run: yarn lint:ci
 
+      - name: Check commit message is valid
```

Comment:
I was going to propose we remove this as it becomes hard when merging across branches. It usually fails the build. Could it be a warning? Please note that we get some random PRs from people and I don't want that this strict rule turns them away from merging a single PR

Reply:
Alright, I will remove that. We can revisit later on if needed 👍